.github/workflows/snyk.yml

```
@@ -94,6 +112,13 @@ jobs:
           export VERSION_TAG=$([[ "${VERSION}" == *-SNAPSHOT ]] && echo 'development' || echo "${VERSION}")
           export LIFECYCLE=$([[ "${VERSION}" == *-SNAPSHOT ]] && echo 'development' || echo 'production')
 
+          # use inputs.dockerImage if present; otherwise if building, use the output of the build step; otherwise infer from version and default repository camunda/zeebe
+          export DOCKER_IMAGE=$([[ ! -z '${{ inputs.dockerImage }}' ]] && echo '${{ inputs.dockerImage }}' || ([[ '${{ inputs.build }}' == 'true' ]] && echo '${{ steps.build-docker.outputs.image }}' || echo "camunda/zeebe:${VERSION}"))
+
+          if [[ '${{ inputs.useMinorVersion }}' == 'true' ]]; then
+            export TARGET="$(echo $VERSION | sed -E -e 's/\.[^.]*$//')"
```

🔧  LGTM, one idea that would be more strict to only catch major and minor, shouldn't matter much though 
```suggestion
            export TARGET="$(echo $VERSION | sed -E -e 's/^([0-9]+\.[0-9]+).*$/\1/')"
```